app/src/main/java/com/bogdan/codeforceswatcher/feature/users/redux/Sorting.kt

```diff
@@ -0,0 +1,21 @@
+package com.bogdan.codeforceswatcher.feature.users.redux
+
+import com.bogdan.codeforceswatcher.model.User
+import com.bogdan.codeforceswatcher.room.DatabaseClient
+
+object Sorting {
+
+    fun sort(users: List<User>, sortType: UsersState.SortType): List<User> =
```

Comment:
`when` is usually placed in the same row as `): List<User>`, which can be removed, I believe.